package/MDAnalysis/analysis/rdf.py

```diff
@@ -645,9 +645,20 @@ def _single_frame(self):
                 backend=self.backend,
             )
 
+            # The following is an optimized version based on the old logic.
```

Comment:
remove this comment because it's not useful for anyone reading the code (what is "old logic"?); if you want to communicate this information to reviewers, comment in GitHub on your code (basically, review yourself)